htcanalyze/htcanalyze.py

```diff
@@ -224,7 +178,7 @@ def log_to_dict(self, file: str, sec: int = 0
         Consider that the return values can be None or empty dictionarys
         """
         job_events = list()
-        res_dict = dict()
+        resource_list = list()
```

Comment:
why not call it `resources` instead of `resource_list`? The list part is a bit redundant.

Reply:
Sure :)